client/cli/go/cmds/node.go

```diff
@@ -265,6 +265,10 @@ var nodeInfoCommand = &cobra.Command{
 		if err != nil {
 			return err
 		}
+		const entryStateRemoved api.EntryState = "removed"
+		if info.State == api.EntryStateFailed {
+			info.State = entryStateRemoved
```

Comment:
This is good, but quick question? Why create a const? Just in case it is used in the future?